lib/SimpleSAML/IdP.php

```diff
@@ -90,13 +90,15 @@ private function __construct(string $id)
             }
             $this->config = $metadata->getMetaDataConfig(substr($id, 5), 'adfs-idp-hosted');
 
+            /*
```

Comment:
This seems misplaced

Reply:
You're right. This is just a left over of a workaround of a bug we discovered.
I will remove it from this PR and raise a new issue to resolve this.

Reply:
The issue has been raised #1564 and the change been reverted.

Reply:
Either remove the commented code or use it...   Or was this something my rebase raised?

Reply:
This must have been raised by the rebase. I removed it earlier.